src/overlays/actors/ovl_Bg_Kin2_Picture/z_bg_kin2_picture.h

```diff
@@ -3,17 +3,29 @@
 
 #include "global.h"
 
+#define BG_KIN2_PICTURE_SKULLTULA_COLLECTED(thisx) (((thisx)->params >> 5) & 1)
+#define BG_KIN2_PICTURE_GETS_3FC(thisx) ((u8)(((thisx & 0x3FC)) >> 2))
```

Comment:
```suggestion
#define BG_KIN2_PICTURE_GET_3FC(thisx) ((u8)(((thisx & 0x3FC)) >> 2))
```

Reply:
Saw GETS in the code base and found it odd, but still went with it for consistency. Glad to know I can go for the more intuitive thing. Thanks!

Reply:
Yeah, only 4 files use it, we need to clean it up.